services/src/main/java/org/exoplatform/webconferencing/rest/RESTWebConferencingService.java

```diff
@@ -0,0 +1,734 @@
+import javax.ws.rs.DELETE;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.CacheControl;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.services.rest.resource.ResourceContainer;
+import org.exoplatform.services.security.ConversationState;
+import org.exoplatform.webconferencing.CallInfo;
+import org.exoplatform.webconferencing.CallInfoException;
+import org.exoplatform.webconferencing.CallState;
+import org.exoplatform.webconferencing.GroupInfo;
+import org.exoplatform.webconferencing.IdentityNotFound;
+import org.exoplatform.webconferencing.InvalidCallStateException;
+import org.exoplatform.webconferencing.UserInfo;
+import org.exoplatform.webconferencing.UserState;
+import org.exoplatform.webconferencing.WebConferencingService;
+import org.exoplatform.webconferencing.client.ErrorInfo;
+
+/**
+ * Created by The eXo Platform SAS.
+ *
+ * @author <a href="mailto:pnedonosko@exoplatform.com">Peter Nedonosko</a>
+ * @version $Id: RESTWebConferencingService.java 00000 Feb 22, 2017 pnedonosko $
+ */
+@Path("/webconferencing")
+@Produces(MediaType.APPLICATION_JSON)
+public class RESTWebConferencingService implements ResourceContainer {
+
+  /** The Constant ME. */
+  public static final String             ME    = "me";
+
+  /** The Constant EMPTY. */
+  public static final String             EMPTY = "".intern();
+
+  /** The Constant LOG. */
+  protected static final Log             LOG   = ExoLogger.getLogger(RESTWebConferencingService.class);
+
+  /** The video calls. */
+  protected final WebConferencingService webConferencing;
+
+  /** The cache control. */
+  private final CacheControl             cacheControl;
+
+  /**
+   * Instantiates a new REST video calls service.
+   *
+   * @param skype
+   *          the skype
+   */
+  public RESTWebConferencingService(WebConferencingService skype) {
+    this.webConferencing = skype;
+    this.cacheControl = new CacheControl();
+    cacheControl.setNoCache(true);
+    cacheControl.setNoStore(true);
+  }
+
+  /**
+   * Gets the user info.
+   *
+   * @param uriInfo
+   *          the uri info
+   * @param userName
+   *          the id
+   * @return the user info response
+   */
+  @GET
+  @RolesAllowed("users")
+  @Path("/user/{name}")
+  public Response getUserInfo(@Context UriInfo uriInfo, @PathParam("name") String userName) {
+    ConversationState convo = ConversationState.getCurrent();
+    if (convo != null) {
+      String currentUserName = convo.getIdentity().getUserId();
+      if (userName != null) {
+        if (ME.equals(userName)) {
```

Comment:
Could be a problem if an user is really named "me". Why do we need this special value ?

Reply:
Ok, it might. Need get rid of this.